bitnami/kubeapps/README.md

```diff
@@ -937,6 +937,34 @@ Feel free to [open an issue](https://github.com/kubeapps/kubeapps/issues/new) if
 
 ## Troubleshooting
 
+### Upgrading to chart version 8.0.0
+
+This major release renames several values in this chart and adds missing features, in order to be inline with the rest of assets in the Bitnami charts repository.
```

Comment:
Suggestion: _This major release renames several values in this chart and adds missing features, in order to get aligned with the rest of the assets in the Bitnami charts repository._